docs/cryptodoc/src/03_mac.rst

```diff
@@ -80,3 +80,33 @@ that the developer sets the nonce before each new GMAC computation.
 
 **Remark:** GMAC is generally used in AES-GCM. For different
 encryption mechanisms HMAC and CMAC should be used in favor of GMAC.
+
+KMAC
+----
```

Comment:
Originally, KMAC was introduced in Botan 3.2.0 but we failed to mention it in the cryptodoc. That may have been by design, I'm not sure anymore.

Nevertheless, now that we mention KMAC in the context of SP800-56Cr2, I guess it's just fair to also mention it explicitly in the MAC section of the cryptodoc.